pages/linux/dnstracer.md

```diff
@@ -0,0 +1,28 @@
+# dnstracer
+
+> The dnstracer command determines where a DNS gets its information from.
+> More information: <https://manned.org/dnstracer>.
+
+- Find out where your local DNS got the information on www.example.com:
+
+`dnstracer {{www.example.com}}`
+
+- Start with a [s]pecific DNS that you already know:
+
+`dnstracer -s {{dns.example.org}} {{www.example.com}}`
+
+- Only query IPv4 servers:
+
+`dnstracer -4 {{www.example.com}}`
+
+- [r]etry each request 5 times on failure:
```

Comment:
Hey, Sorry for the request can you revert this change?
```suggestion
- Retry each request 5 times on failure:
```
As `r` means `retries` your original version of the command description is fine as the mnemonic wouldn't be applicable here.

Reply:
Of course, makes sense. Personally, I'd be fine with either, but it's of course preferable to be consistent with other pages.